src/Compilers/CSharp/Portable/Generated/CSharpSyntaxGenerator/CSharpSyntaxGenerator.SourceGenerator/Syntax.xml.Internal.Generated.cs

```diff
@@ -45692,248 +45692,5 @@ public static NullableDirectiveTriviaSyntax NullableDirectiveTrivia(SyntaxToken
 
             return new NullableDirectiveTriviaSyntax(SyntaxKind.NullableDirectiveTrivia, hashToken, nullableKeyword, settingToken, targetToken, endOfDirectiveToken, isActive);
         }
-
-        internal static IEnumerable<Type> GetNodeTypes()
```

Comment:
>GetNodeTypes [](http://example.com/codeflow?start=42&length=12)

Can we remove the corresponding method from `Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.SyntaxFactory` as well?